### ISSUE TITLE
Request - media license - copyright restricted - better definition?

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8623]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

Post-https://github.com/ArctosDB/arctos/issues/8370, there are over half a million records using https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_license#copyright_restricted, and I don't think the current definition is great.

>License is restricted to the copyright holder (see Comment). Arctos insitution cannot license the media.

* there's not necessarily any comment
* "Arctos insitution" may not be the involved party, and the license may have been "chosen" for any number of reasons.

Please help form a better definition. It's terrible, but I'll propose:

>Usage is restricted.

as a starting point.

(I'm also not crazy about the URL - https://www.loc.gov/duplicationservices/copyright/ - but it gets the idea across.)

HELP!

**Comment by DellaCHall** [comment]
Not as short as your suggestion @dustymc, but what about: "Owner of the intellectual property has the exclusive legal right to reproduce, distribute copies, create derivatives or adaptations, perform, and display the creative work."

Could also add something like "Contact owner for licensing."

 And link to this URL? https://www.copyright.gov/

**Comment by droberts49** [comment]
I would combine it to make it clear and cover the bases: 
Use is restricted. Owner of the intellectual property has the exclusive legal right to reproduce, distribute copies, create derivatives or adaptations, perform, and display the creative work. Contact intellectual property owner for licensing.

Linking to the US Copyright office is good. You could provide a link to Title 17: https://www.copyright.gov/title17/

**Comment by dustymc** [comment]
>provide a link 

There's one "data link" (defines the license) required for this table, but any number of "documentation links" can be added as well. 

Possibly a less US-centric view would be useful, if that doesn't conflict with US needs. https://www.wipo.int/en/web/copyright?? https://copyrightresource.uw.edu/copyright-law/international-copyright-law/? ???
